src/electron/linux-sign-verify/linux-sign-verify.ts

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import * as process from 'child_process';
+import { autoUpdater } from 'electron-updater';
+import { isNil } from 'lodash';
+import * as os from 'os';
+
+const verifySignatureCommand = function(verifyCommand, callback): void {
+    process.exec(verifyCommand, (err, stderr, stdout) => {
+        if (!isNil(err)) {
+            console.log("There's an error:", err);
+            return;
+        } else {
+            if (!isNil(stdout)) {
+                console.log('stdout:\n', stdout);
+                callback(stdout);
+            } else if (!isNil(stderr)) {
+                console.log('stderr:\n', stderr);
+                callback(stderr);
+            } else {
+                return;
+            }
+        }
+    });
+};
+
+const downloadFilesCommand = function(downloadCommand, callback): void {
+    process.exec(downloadCommand, _ => {
+        callback();
+    });
+};
+
+export function verifySignatureOnLinux(): void {
+    if (os.platform() !== 'linux') {
+        return;
+    }
+
+    const downloadCommand =
+        'mkdir verifyDetachedSignatureForAIAndroid\n' +
+        'wget -O "verifyDetachedSignatureForAIAndroid/doc.AppImage" "https://a11yinsightsandroidblob.blob.core.windows.net/aimobile-canary/Accessibility Insights for Android.AppImage"\n' +
+        'wget -O "verifyDetachedSignatureForAIAndroid/doc.sig" "https://a11yinsightsandroidblob.blob.core.windows.net/aimobile-canary/Accessibility Insights for Android.sig"\n';
```

Comment:
I don't think we want to hard code this URL since we will have multiple in the near future. We can access the root URL using appUpdater.getFeedURL() and then append on the file name

Reply:
appUpdater.getFeedURL(), isn't working anymore
![image](https://user-images.githubusercontent.com/45858632/71858140-fca72580-309e-11ea-89ee-a47e03e79cc9.png)